hopli/src/key_pair.rs

```diff
@@ -39,6 +59,33 @@ pub fn read_identities(files: Vec<PathBuf>, password: &str) -> Result<HashMap<St
     Ok(results)
 }
 
+pub fn update_identity_password(
+    keys: HoprKeys,
+    path: &PathBuf,
+    password: &str,
+) -> Result<Option<(String, HoprKeys)>, HelperErrors> {
+    let file_path = path
+        .to_str()
+        .ok_or(HelperErrors::IncorrectFilename(path.to_string_lossy().to_string()))?;
+
+    if file_path.ends_with(".id") {
```

Comment:
The check should be: exists and is a file. Also, what happens, if the entire path does not exist? The dir structure is not created?

Reply:
Nope. If the file is not found, nothing should happen